src/SonarScanner.MSBuild.PreProcessor/WebServer/SonarWebServer.cs

```diff
@@ -163,6 +158,22 @@ public async Task<JreMetadata> DownloadJreMetadataAsync(string operatingSystem,
         }
     }
 
+    public async Task<EngineMetadata> DownloadEngineMetadataAsync()
+    {
+        const string api = "analysis/engine";
```

Comment:
```suggestion
        var api = "analysis/engine";
```
[This creates the same IL](https://stackoverflow.com/a/49224799), so there is no benefit (except immutability) to use `const` here.

Reply:
Which is non-sonarway and imho should be decommissioned. I'm not marking local variables as immutable until C# gives me the `let` keyword or something similar succinct.